src/commands/new-tip/index/processor.ts

```diff
@@ -495,8 +495,7 @@ export async function executeTip(
   if (moniker) {
     const monikerVal = moniker as ResponseMonikerConfigData
     const amountMoniker = roundFloatNumber(
-      payload.amount[0] /
-        (payload.recipients.length * (monikerVal?.moniker?.amount || 1)),
+      payload.amount[0] / (monikerVal?.moniker?.amount || 1),
```

Comment:
you should fix the amount from this `getTipPayload (L300-304)`
https://github.com/consolelabs/mochi-discord/pull/965/files#diff-d3777f99f6c2deebdcbd7212afea884e8a9d16d1db91e17005ebf8a7314c32dbL300-L304

Reply:
chỗ kia vẫn đúng ạ, chỉ là công thức lúc tính amountMoniker bị sai nên phải sửa lại